go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/edaniels/golinters
+module github.com/mongodb-forks/golinters
 
 go 1.13
```

Comment:
do we need to bump the go version up as well?

Reply:
sure, I'll bump this to 1.17